build/android/src/main/res/layout/assetcopy.xml

```diff
@@ -18,7 +18,7 @@
         android:ellipsize="middle" 
         android:singleLine="true"
         android:layout_gravity="center_horizontal"
-        android:text="preparing media ..."
+        android:text="@string/preparing_media"
```

Comment:
A bit more than that.  And perhaps it doesn't have to be translated, but it will throw a lint warning, and there's not really a good reason to not do it the "right" way.